llvm-project/README.md

```diff
@@ -0,0 +1,65 @@
+# llvm-project
+
+## Goals
+To produce statically-linked builds of Clang and LLVM utilities supercharged
+for Linux kernel development, from a series of Docker images built
+continuously, that can be hosted on
+[kernel.org](https://mirrors.edge.kernel.org/pub/tools/crosstool/).
+
+### Stage N goal (stage 5)
```

Comment:
It may be easier to read with these in chronological order.

Reply:
done PTAL